content/docs/BestPractices/CustomTelemetry/index.md

```diff
@@ -71,7 +71,7 @@ Of course it is possible to have a single object as a central place to emit tele
 
 ## Candidate data for telemetry
 
-Telemetry must be **actionable** for the customer. Do not emit signals that they cannot act on (knowing about CPU performance counters on the database is useless if the partner cannot scale the database)
+Telemetry must be **actionable** for the customer. Do not emit signals that they cannot act on period (knowing about CPU performance counters on the database is useless if the partner cannot scale the database).
```

Comment:
```suggestion
Telemetry must be **actionable** for the customer. Do not emit signals that they cannot act on (knowing about CPU performance counters on the database is useless if the partner cannot scale the database).
```

Reply:
@JesperSchulz can you please enlighten me what you meant by https://github.com/microsoft/alguidelines/pull/142#discussion_r824610748? thanks in advance